dm_cli/utils.py

```diff
@@ -115,3 +120,39 @@ def concat_dependencies(
             raise ApplicationException(f"Conflicting dependency alias(es) in file '{filename}'. '{alias_intersect}'")
     old_dependencies.update(entity_dependencies)
     return old_dependencies
+
+
+def unpack_and_save_zipfile(overwrite: bool, export_location: str, zip_file: ZipFile):
+    """Unpack zipfile and save it to export_location. It is assumed that zip file only contains json files and folders.
+    If overwrite is True, any existing file/folder with the name of filename will be overwritten.
+    """
+    if not zip_file.filename.endswith(".zip"):
+        raise ApplicationException(message="file ending .zip must be included in filename!")
+    zip_file_unpacked_path = f"{export_location}/{zip_file.filename.rstrip('.zip')}"
+
```

Comment:
This should fix that TODO:
```python
if len(zip_file.filelist) == 1:
    zip_file_unpack_path = zip_file_unpacked_path.parent
```

Reply:
nice, found a slightly different method since I had to handle the extraction  of the zip file a bit more carefully.